ui/src/app/edge/settings/channels/channels.component.html

```diff
@@ -23,12 +24,13 @@
               </ion-item>
               <ion-item lines="none">
                 <ion-select style="white-space: initial;width: 100%;" #selectedChannelId interface="alert"
+                  [disabled]="!componentSelected"
                   [placeholder]="'CHANNELS.CHANNEL'| translate">
-                  <ng-container *ngIf="selectedComponentId && selectedComponentId.value">
+                  <ng-container *ngIf="componentSelected">
```

Comment:
you could have also used ```selectedComponentId``` to disable the channel button

Reply:
You are right, fixed this Line